contractsV5/IKyberTradeLogic.sol

```diff
@@ -0,0 +1,42 @@
+pragma  solidity 0.5.11;
+
+import "./IKyberReserve.sol";
+import "./IKyberHint.sol";
+
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////
+/// @title Kyber Network main contract
+interface IKyberTradeLogic {
+
+    enum ResultIndex {
+         tradeWei,
+         networkFeeWei,
+         platformFeeWei,
+         rateWithNetworkFee,
+         numFeePayingReserves,
+         feePayingReservesBps,
+         destAmountNoFee,
+         actualDestAmount,
+         destAmountWithNetworkFee,
+         last
+    }
+    
+    enum FeesIndex {
+        takerFee,
+        customFee
+    }
+    
+    function addReserve(address reserve, uint reserveId, bool isFeePaying) external returns(bool);
```

Comment:
I think `addReserve` and `listPairForReserve` should not be part of the KyberTradeLogic contract, since as its name suggests, should only have the trade logic.

Reply:
trade logic must have this data for running trade logic.
I am thinking to network will call trade logic.
so we don't need a seperate call.
also notice API here is a bit different. un needed data isn't sent here.

regarding list pair
I am also thinking it should call this API (so we don't need separate call)
I also think listing of token per reserve will not be done in network. but only here.